benchmark/arrayops/arrayops.d

```diff
@@ -3,7 +3,7 @@
  *
  * Copyright: Copyright Martin Nowak 2016 -.
  * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
- * Authors:    Martin Nowak
```

Comment:
Made a fix for github's language detection.
[better heuristic detection of D Programming Language by MartinNowak · Pull Request #3145 · github/linguist](https://github.com/github/linguist/pull/3145)